packages/ndla-icons/src/editor/Superscript.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * Copyright (c) 2019-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+// N.B! AUTOGENERATED FILE. DO NOT EDIT
```

Comment:
autogenereres den med 2019?

Reply:
Jepp.

Reply:
Det baserer seg på fila COPYRIGHT som har 2019. Kan generere på nytt men då får alle 2019.

Reply:
ser forrige commit i den filen var "chore: update copyright year for icons" så er kanskje det som er meningen? Syns det er litt rart da å skulle oppdatere alle pga det.